src/NuGet.Core/NuGet.Commands/RestoreCommand/LockFileBuilder.cs

```diff
@@ -243,8 +257,12 @@ public class LockFileBuilder
                                 targetFrameworkOverride: nonFallbackFramework,
                                 dependencyType: includeFlags,
                                 dependencies: graphItem.Data.Dependencies,
-                                cache: builderCache);
+                                cache: builderCache,
+                                maccatalystFallback: maccatalystFallback);
 
+                            // NOTE: 1701 goes here, but we don't have enough
```

Comment:
Are you referring to NuGet log code ?

```suggestion
                            // NOTE: NU1701 goes here, but we don't have enough
```